{
  "Launchpad": "https://bugs.launchpad.net/bugs/1371677",
  "Id": "1371677",
  "Gerrit": "https://review.openstack.org/174836",
  "IdGerrit": "174836",
  "IdCommit": "833357301bc80a27422f7bf081fae2d3da730a24",
  "Files": [
    "nova/tests/unit/virt/libvirt/test_driver.py",
    "nova/virt/libvirt/driver.py"
  ],
  "Lines_Inseted": 6,
  "Lines_Deleted": 1,
  "CommitParent": "d1fb8d0fbdd6cb95c43b02f754409f1c728e8cd0",
  "Classification": "Bug",
  "KeywordsTitle": "test will fail, failure, failed test,",
  "Description": "test will fail, failure, failed test,",
  "KeywordsCommit": "",
  "Comments": "This is a partial bug"
}